AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/CampaignManagerInviteController.cs

```diff
@@ -0,0 +1,113 @@
+﻿using AllReady.Areas.Admin.Features.CampaignManagerInvites;
+using AllReady.Areas.Admin.ViewModels.ManagerInvite;
+using AllReady.Features.Campaigns;
+using AllReady.Models;
+using AllReady.Security;
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace AllReady.Areas.Admin.Controllers
+{
+    [Area("Admin")]
+    [Authorize(nameof(UserType.OrgAdmin))]
+    public class CampaignManagerInviteController : Controller
+    {
+        private readonly IMediator _mediator;
+        private readonly UserManager<ApplicationUser> _userManager;
+
+        public CampaignManagerInviteController(IMediator mediator, UserManager<ApplicationUser> userManager)
+        {
+            _mediator = mediator;
+            _userManager = userManager;
+        }
+
+        // GET: Admin/Invite/Send/5
+        [Route("[area]/[controller]/[action]/{campaignId:int}")]
+        public async Task<IActionResult> Send(int campaignId)
+        {
+            var campaign = await _mediator.SendAsync(new CampaignByCampaignIdQuery { CampaignId = campaignId });
+            if (campaign == null) return NotFound();
```

Comment:
I'll echo the same thing I reviewed on `EventManagerInviteController`, `CampaignByCampaignIdQueryHandler` is loading a lot of data to return an entire `Campaign`.

All you need it enough data to build your ViewModel:
```
return View("Send", new CampaignManagerInviteViewModel
{
    CampaignId = campaignId,
    CampaignName = campaign.Name,
});
```

I'd write a new query and handler called CampaignManagerInviteQuery and just query for the data you need, and let the handler build and return CampaignManagerInviteViewModel